sdk/template/template/test/public/configurationClient.spec.ts

```diff
@@ -98,7 +102,7 @@ describe("[AAD] ConfigurationClient functional tests", function () {
     // The supportsTracing assertion from chaiAzure can be used to verify that
```

Comment:
Just outside of the diff here there's a TODO comment. Can we remove that now or is it still relevant?

Reply:
It's somewhat relevant here.  Will revisit soon.